app/assets/javascripts/discourse/app/components/composer-editor.js

```diff
@@ -487,24 +488,27 @@ export default Component.extend(ComposerUploadUppy, {
 
   _warnMentionedGroups(preview) {
```

Comment:
btw. should this function (and `_warnHereMention()`) be debounced the same way as `_warnCannotSeeMention()`?